tracks/python/exercises/square-root/mentoring.md

```diff
@@ -0,0 +1,26 @@
+# Mentoring
+Consider guiding towards a solution in which none of the existing Python library functions are used, but rather one in which students create the logic/algorithm themselves.
```

Comment:
```suggestion
# Mentoring

Consider guiding towards a solution in which none of the existing Python library functions are used, but rather one in which students create the logic/algorithm themselves.
```

Is there a particular library/function that should be avoided? Why should it be avoided? This is rather vague and I'm not sure how mentors are supposed to use it.